docs/how-to/simple-authpolicy-for-app-developers.md

```diff
@@ -0,0 +1,222 @@
+# Authenticated API for Application Developers
```

Comment:
Have simplified this quite a bit while we wait for AuthPolicy v1beta2 to land (basically - it just looks at auth now)

Reply:
One other option is we park this for now, and ship a more well-rounded v1beta1 authn/z guide